CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.3.1 (Unreleased)
+
+Features:
+
+- Fix authentication header on REE-1.8.7-2011.03
```

Comment:
This would apply to all ruby 1.8.7 implementations (in theory), right? Not just REE.